### PR TITLE
Add new Artist brushes

### PR DESCRIPTION
**Pull request by maddiedierker**
Adding more brushes to Artist as part of Hack Days:

Screen Shot 2020-05-11 at 9 58 36 AM

## Links

- [Hack Days doc](https://docs.google.com/document/d/1W-5Uh94GZZDOgtuud4FvAkBztLh-_yyQxXh0TKf228w/edit#bookmark=id.dqfey4rkeump)

## Testing story

Tested manually.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by maddiedierker** [comment]
cc @markabarrett here are the new brushes! 🎉

File: apps/src/turtle/linePatterns.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+exports.load = function(assetUrl) {
+  return {
+    patternDefault: assetUrl('media/common_images/defaultline-menuicon.png'), // default first, then alphabetical
```
Comment (jmkulwik) [inline apps/src/turtle/linePatterns.js:3]: Thank you for this comment 👍

**Comment by jmkulwik** [comment]
Can you do a quick test on mobile (can be sauce labs) to see how these will display vs. how they displayed when the list was smaller?

**Comment by maddiedierker** [comment]
good call, jessie! we actually removed a bunch of the new brushes we wanted to add to make sure the dropdown wasn't too wide on mobile. screenshot from iOS (Android looks about the same) in SauceLabs:
Screen Shot 2020-05-13 at 11 45 01 AM

**Review by jmkulwik — APPROVED** [review]
🎉

**Pull request merged**